ClientKit/ClientReportTypes.cs

```diff
@@ -20,6 +20,15 @@
 using System.Runtime.InteropServices;
 using OSVR.ClientKit;
 
+using ChannelCount = System.Int32;

```

Comment:
Can we rename this to SensorCount? See https://github.com/OSVR/OSVR-Core/issues/166